.github/workflows/node-test.yaml

```diff
@@ -127,11 +132,31 @@ jobs:
       uses: ./.github/tmp/post-install-steps
 
 
+    - name: Set environment variables for tests
+      id: set-env-vars-for-tests
+      shell: bash
```

Comment:
What happens on a windows test run?

Reply:
It uses `bash` on Windows and works: https://github.com/dominykas/pkgjs-action/runs/4684547350?check_suite_focus=true

Reply:
Ah i see, because it’s just used to inform GHA, not to change anything about the runner. Nice!

This one step along seems like it’d be useful as a reusable action :-)